airflow/bin/cli.py

```diff
@@ -72,15 +72,30 @@ def sigquit_handler(sig, frame):
     print("\n".join(code))
 
 
-def setup_logging(filename):
-    root = logging.getLogger()
+def setup_file_logging(filename, fmt=settings.SIMPLE_LOG_FORMAT):
     handler = logging.FileHandler(filename)
```

Comment:
@bolkedebruin  I felt like that would be presenting @mtdewulf's work as my own, but I can certainly do that if preferred (indeed, I have done in order to begin working on this).